notebooks/276-mobilevlm-language-assistant/276-mobilevlm-language-assistant.ipynb

```diff
@@ -0,0 +1,784 @@
+{
```

Comment:
<p>could you please provie link on MobileLLaMA?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/1627/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Reply:
It was introduced in the same paper. I added a comment about it in the header section.

Reply:
I mean link on model card from hf hub